lib/core/presentation/pages/event/event_detail_page/helper/event_detail_navigation_bar_helper.dart

```diff
@@ -55,9 +60,22 @@ class EventDetailNavigationBarHelper {
         ),
         onTap: () {
           Vibrate.feedback(FeedbackType.light);
+          final userId = AuthUtils.getUserId(context);
+          final assignedToMeTicket =
+              context.watch<GetMyTicketsBloc>().state.maybeWhen(
```

Comment:
Have you tested here ? Be careful when using context.watch inside the callback. 
Are we sure that the context contains GetMyTicketsBloc ? 
Suggest passing the tickets through function's params